README.md

```diff
@@ -79,9 +78,33 @@ sudo /usr/local/share/gcm-core/uninstall.sh
 
 ---
 
+### Linux Debian package (.deb)
+
+Download the latest [.deb package](https://github.com/microsoft/Git-Credential-Manager-Core/releases/latest), and run the following:
+
+```shell
+sudo dpkg -i <path-to-package>
+git-credential-manager-core configure
+```
+
+Additional configuration is required on Linux distributions; see [here](https://aka.ms/gcmcore-linuxcredstores).
```

Comment:
I prefer to make our links more helpful, instead of "here". Perhaps...

```
Some Linux distributions require [additional configuration instructions](https://aka.ms/gcmcore-linuxcredstores).
```